ChallengesWithTestsMarkVII/ChallengesSet01.cs

```diff
@@ -8,37 +8,38 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            return num1 == num2;
         }
 
         public double Subtract(double minuend, double subtrahend)
         {
-            throw new NotImplementedException();
+            return minuend - subtrahend;
         }
 
         public int Add(int number1, int number2)
         {
-            throw new NotImplementedException();
+            return number1 + number2;
         }
 
         public int GetSmallestNumber(int number1, int number2)
         {
-            throw new NotImplementedException();
+            return Math.Min(number1, number2);
         }
 
         public long Multiply(long factor1, long factor2)
         {
-            throw new NotImplementedException();
+            return factor1 * factor2;
         }
 
         public string GetGreeting(string nameOfPerson)
         {
-            throw new NotImplementedException();
+            nameOfPerson = Console.ReadLine();
```

Comment:
We don't want a `Console.ReadLine()` in this method.  We've used these a number of times during class to get the user's input, but for this method we're assuming that the name will be given to us as an argument (`nameOfPerson`) supplied by the caller of the method

Reply:
Also, I don't believe this will pass the tests as-written.